TestingHomework-Discounts/DTOS/CartDTO.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using System.Text;
+using TestingHomework.Tests.DTOs;
+using TestingHomework_Discounts;
+
+namespace TestingHomework.Tests.Data
+{
+    [Table("Cart")]
+    public class CartDTO : IDatabaseObjectBase
+    {
+        [Key, DatabaseGenerated(DatabaseGeneratedOption.Identity)]
```

Comment:
This DTO can't possibly function. It does not include all information required for a cart